purchases/src/test/java/com/revenuecat/purchases/teststore/TestStoreProductConverterTest.kt

```diff
@@ -72,10 +75,10 @@ class TestStoreProductConverterTest {
             )
         )
 
-        val result = TestStoreProductConverter.convertToStoreProduct(productResponse)
+        val result = convertToStoreProduct(productResponse)
```

Comment:
Now that the `locale` is a parameter of the `convertToStoreProduct` method, perhaps we should pass the locale here in the test to make them deterministic? (the same in the other tests)

Reply:
Ohhh, didn't realize that. Sorry! Then it's perfect. Thanks for clarifying! 🙏